### DISCUSSION TITLE
Issue Running soup with Minimal Docker Containers on Security Onion

### DISCUSSION THREAD

**Discussion by SliuzasLukas** [Security-Onion-Solutions/securityonion#13544] [Unsupported Versions]
<html>
<body>
<p>Hi everyone,</p><p>I'm running Security Onion with a minimal set of Docker containers because I'm forwarding logs to a separate Elasticsearch instance. However, when I attempt to run <code>soup</code>, I encounter the following error:</p><pre><div class="dark bg-gray-950 rounded-md border-[0.5px] border-token-border-medium"><div class="flex items-center relative text-token-text-secondary bg-token-main-surface-secondary px-4 py-2 text-xs font-sans justify-between rounded-t-md"><span>csharp</span><div class="flex items-center"><span class="" data-state="closed"><button class="flex gap-1 items-center"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" viewBox="0 0 24 24" class="icon-sm"><path fill="currentColor" fill-rule="evenodd" d="M7 5a3 3 0 0 1 3-3h9a3 3 0 0 1 3 3v9a3 3 0 0 1-3 3h-2v2a3 3 0 0 1-3 3H5a3 3 0 0 1-3-3v-9a3 3 0 0 1 3-3h2zm2 2h5a3 3 0 0 1 3 3v5h2a1 1 0 0 0 1-1V5a1 1 0 0 0-1-1h-9a1 1 0 0 0-1 1zM5 9a1 1 0 0 0-1 1v9a1 1 0 0 0 1 1h9a1 1 0 0 0 1-1v-9a1 1 0 0 0-1-1z" clip-rule="evenodd"></path></svg>Copy code</button></span></div></div><div class="overflow-y-auto p-4" dir="ltr"><code class="!whitespace-pre hljs language-csharp">Starting crond service at <span class="hljs-number">07</span>:<span class="hljs-number">47</span>:<span class="hljs-number">38.538922</span>
Successfully started crond.

Starting salt-master service at <span class="hljs-number">07</span>:<span class="hljs-number">47</span>:<span class="hljs-number">38.568496</span>
Successfully started salt-master.

Starting salt-minion service at <span class="hljs-number">07</span>:<span class="hljs-number">47</span>:<span class="hljs-number">39.666973</span>
Successfully started salt-minion.

Enabling highstate.
[<span class="hljs-meta">INFO    </span>] Creating module dir <span class="hljs-string">'/var/cache/salt/minion/extmods/grains'</span>
[<span class="hljs-meta">INFO    </span>] Syncing grains <span class="hljs-keyword">for</span> environment <span class="hljs-string">'base'</span>
[<span class="hljs-meta">INFO    </span>] Loading cache <span class="hljs-keyword">from</span> salt:<span class="hljs-comment">//_grains, for base</span>
[<span class="hljs-meta">INFO    </span>] Caching directory <span class="hljs-string">'_grains/'</span> <span class="hljs-keyword">for</span> environment <span class="hljs-string">'base'</span>
local:
    ----------
    msg:
        Info: highstate state enabled.
    res:
        True

Soup failed <span class="hljs-keyword">with</span> error <span class="hljs-number">7</span>: Unhandled error
Unhandled error occurred, please check /root/soup.log <span class="hljs-keyword">for</span> details.
</code></div></div></pre><p><strong>Current running Docker containers:</strong></p>
Container | Status | Details
-- | -- | --
so-dockerregistry | running | Up 8 minutes
so-idstools | running | Up 10 days
so-influxdb | running | Up 10 days (healthy)
so-kratos | running | Up 10 days
so-nginx | running | Up 10 days (healthy)
so-sensoroni | running | Up 10 days
so-soc | running | Up 10 days

<p><strong>My question:</strong>
Which additional Docker containers do I need to have running to successfully execute <code>soup</code> and update Security Onion in this minimal setup?</p><p>I've checked the <code>/root/soup.log</code> but didn't find clear indications of what's missing or causing the error. Any guidance or recommendations would be greatly appreciated!</p><p>Thanks in advance for your help!</p>
</body>
</html>

**Comment by cm-ops** [comment]
Sounds like you have modified SO considerably. That would cause soup to fail.

**Reply by SliuzasLukas** [reply]
Currently only modification I have done is disabled dockers - set status to false
